eventually/team/models.py

```diff
@@ -31,37 +32,74 @@ class Team(models.Model):
         :param update_date: The date when
                     the certain team was last time edited.
         :type: create_date: datetime
+
+        :param owner: Describing relation between team and owner of team.
+        :type owner: int
+
+        :param members: Describing
+                            relations between team and members of team.
+        :type members: list
     """
     name = models.CharField(max_length=30)
     description = models.CharField(max_length=1024, blank=True)
     image = models.CharField(max_length=300, blank=True)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
+    owner = models.ForeignKey(CustomUser, null=True)
+    members = models.ManyToManyField(CustomUser, related_name='+')
 
     def __str__(self):
         """
         Magic method that returns string representation of
         team instance object.
 
-        :return: team name, team description
+        :return: string with team's information
+        """
+
+        return "{} {} {} {} {} {} {} {}".format(self.id,
+                                                self.name,
+                                                self.description,
+                                                self.image,
+                                                self.created_at,
+                                                self.updated_at,
+                                                self.owner.id if self.owner else None,
+                                                [members.id for members \
+                                                in self.members.all()] \
+                                                if self.members else None)
+
+    def __repr__(self):
+        """
+        Magic method that returns representation of
+        team instance object.
+
+        :return: string with team's information
         """
 
-        return "{} {}".format(self.name, self.description)
+        return "{} {} {} {} {} {}".format(self.id,
+                                          self.name,
+                                          self.description,
+                                          self.image,
+                                          self.owner.id if self.owner else None,
+                                          [members.id for members \
+                                          in self.members.all()] \
+                                          if self.members else None)
 
     def to_dict(self):
         """
         Method that converts team object to dictionary.
 
-        :return: dictionaty with team's information
+        :return: dictionary with team's information
 
         :Example:
         {
         'id': 1,
-        'name': 'name',
+        'name': 'name',feature/team
         'description': 'desc',
```

Comment:
maybe it is a good idea to write each dict's field from a new line using `|` char.